src/pages/deployer/index.tsx

```diff
@@ -158,43 +213,69 @@ function Description() {
+        . This free educational tool allows you to deploy your own Jetton to mainnet in one click.
+        You will need at least 0.25 TON for deployment fees. <br />
         <Spacer />
         For detailed instructions and in-depth explanations of all fields please see the{" "}
-        <Link
-          target="_blank"
-          href="https://github.com/ton-defi-org/jetton-deployer-contracts#jetton-metadata-field-best-practices">
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter-contract">
           GitHub README
         </Link>
         . It includes several best practice recommendations so please take a look.
         <Spacer />
         Never deploy code that you've never seen before! This deployer is fully open source with all
         smart contract code{" "}
-        <Link target="_blank" href="https://github.com/ton-defi-org/jetton-deployer-contracts">
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter-contract">
           available here
         </Link>
         . The HTML form is also{" "}
-        <Link target="_blank" href="https://github.com/ton-defi-org/jetton-deployer-webclient">
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter">
           open source
         </Link>{" "}
         and served from{" "}
-        <Link target="_blank" href="https://ton-defi-org.github.io/jetton-deployer-webclient">
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter">
           GitHub Pages
         </Link>
         . <Spacer />
-        Is this deployer safe? Yes! Read{" "}
-        <Link
-          target="_blank"
-          href="https://github.com/ton-defi-org/jetton-deployer-contracts#protect-yourself-and-your-users">
-          this
+        Is this deployer safe? Yes!{" "}
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter-contract">
+          Read
         </Link>{" "}
-        to understand why.
+        this to understand.
       </Typography>
+      <IconButton
+        className="github-icon"
+        sx={{ padding: 0, mt: 2 }}
+        href={JETTON_DEPLOYER_CONTRACTS_GITHUB}
+        target="_blank">
+        <StyledGithubIcon width={20} height={20} src={githubIcon} />
+        <Typography
+          ml={1}
+          variant="h5"
+          sx={{
+            color: "#000",
+            fontWeight: 700,
+            fontSize: 16,
+            display: "flex",
+            alignItems: "center",
+          }}>
+          GitHub Repo{" "}
+        </Typography>
+        <Box
+          ml={1}
+          sx={{
+            display: "flex",
+            alignItems: "center",
+            position: "relative",
+          }}>
+          <img
+            width={14}
+            height={14}
+            style={{ position: "absolute", top: "-6px" }}
+            src={rightArrow}
+            alt="Icon"
+          />
+        </Box>
+      </IconButton>
```

Comment:
consider extracting to its own component

Reply:
done